tests/containers/docker_image.pm

```diff
@@ -15,42 +15,38 @@
 # - run some zypper commands
 # - commit the image
 # - remove the container, run it again and verify that the new image works
-# Maintainer: Pavel Dostál <pdostal@suse.cz>
+# Maintainer: Pavel Dostál <pdostal@suse.cz>, qa-c team <qa-c@suse.de>
 
-use base 'consoletest';
+use Mojo::Base qw(consoletest);
 use testapi;
 use utils;
-use strict;
-use warnings;
 use containers::common;
 use containers::container_images;
 use suse_container_urls 'get_suse_container_urls';
-use version_utils qw(is_sle is_opensuse is_tumbleweed is_leap is_public_cloud);
+use scheduler 'get_test_suite_data';
 
 sub run {
-    my ($self) = @_;
-    $self->select_serial_terminal;
-
     my ($image_names, $stable_names) = get_suse_container_urls();
+    my $test_data           = get_test_suite_data();
+    my $container_mgmt_tool = "docker";
```

Comment:
is this really needed? This module is called "docker". I would imagine this is useful if we had a  module which is used for both docker and podman, but in the case we have, everything is docker inside this .pm file.

Reply:
yeah, I like the idea of making things generic when it makes sense, but then we still have specific functions in this module like `install_docker_when_needed` and `scc_apply_docker_image_credentials `

Reply:
Maybe not for this PR, but an idea that comes to my mind out of this discussion is to have `docker_image` and `podman_image` as a single file with your idea of `$container_mgmt_tool` variable, because the logic of both modules are basically the same.

Reply:
i noticed too. But i did not want to do much refactoring on this PR. There was also a [comment](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11166/files#r505296504) in the previous PR to approach things in OO. I would like the idea but i do not know how far you want to go. Let s discuss it outside the ticket

Reply:
Sure, we can do this in another PR.

Reply:
I would keep `'docker'` until we decide to merge the `docker_image.pm` and `podman_image.pm` together.

Reply:
Maybe i do not understand this correctly. The string docker appears in many statements in the file. So i assigned it to a variable. is the problem the name of the variable?

Reply:
Probably yes. Maybe use `$runtime` instead of `$container_mgmt_tool` so it is consistent.

Reply:
Done